### DISCUSSION TITLE
Does godot have a plan to support Minigame? Such as wechat Tikto...

### DISCUSSION THREAD

**Discussion by XiaoFanJx2** [godotengine/godot-proposals#7681] [Platforms]
In China Wechat Tikto ( and more...) Minigame is very popular. In the future, unity China( (https://unity.cn/tuanjie/wxminigame) ) also plans to support minigame, which may be a very big market. At present, we are using cocos create(https://www.cocos.com/creator?_t=t ), but godot is very suitable for minigame. Do you have more support plans?

**Comment by yangqingming** [comment]
+1

**Comment by yangqingming** [comment]
+1

**Comment by AThousandShips** [comment]
@yangqingming Please don't bump without contributing significant new information. Use the :+1: reaction button on the first post instead.
